src/blog/understanding-ecmascript-part-3.md

```diff
@@ -325,14 +325,16 @@ This kind of inference with ASI was deemed too confusing, so static semantics we
 
 There's also another related rule:
 
-> `BindingIdentifier : Identifier`
+> <pre><code class="language-grammar">BindingIdentifier : Identifier</code></pre>
 >
 > It is a Syntax Error if this production has an <code><sub>[Await]</sub></code> parameter and `StringValue` of `Identifier` is `"await"`.
 
-This might be confusing at first. `Identifier` is defined like this:
+This might be confusing at first. [`Identifier`](https://tc39.es/ecma262/#prod-Identifier) is defined like this:
 
-> [`Identifier :`](https://tc39.es/ecma262/#prod-Identifier)
-> `IdentifierName` but not `ReservedWord`
+<!-- markdownlint-disable no-inline-html -->
+<pre><code class="language-grammar">Identifier :
+  IdentifierName <span style="font-style: normal">but not</span> ReservedWord</code></pre>
```

Comment:
```suggestion
<pre><code class="language-grammar">Identifier :
  IdentifierName <b>but not</b> ReservedWord</code></pre>
```

In case you wanna follow the precedent in https://v8.dev/docs/torque#explicit-parameters

Reply:
I used `<b>` for literals, but thought that a human text should be highlighted differently. What is the corresponding human text in the linked snippet for comparison?

Reply:
I guess I'm merging as-is for now.